.github/workflows/ci.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - master
+      - upgrade-sqlx
```

Comment:
I think this implementation is better:
````yaml
on:
  pull_request:
  push:
    branches:
      - master
   ````

Reply:
You're right, I forgot to remove it